docs/src/main/sphinx/connector/iceberg.md

```diff
@@ -176,6 +176,10 @@ implementation is used:
     creation of more data files, since it uses the append operation to insert
     the new records.
   - `true`
+* - `iceberg.per-query-manifest-cache.max-size`
```

Comment:
❤️ 

I'm looking very much forward for cross-query manifest cache functionality.